src/main/java/dude/commands/Interface.java

```diff
@@ -36,9 +36,10 @@ public static void listFullMessage() {
         System.out.println(Interface.LINE);
     }
 
-    public static void addedMessage(String input) {
+    public static void addedMessage(int index) {
         System.out.println(Interface.LINE);
-        System.out.println("added: " + input);
+        System.out.println("added: " + ListManager.getString(index));
+        System.out.println("Now you have " + (index + 1)+ " tasks in the list");
```

Comment:
coding standard: inconsistent spacing after `(index + 1)+ "tasks`
`System.out.println("Now you have " + (index + 1) + " tasks in the list");`